src/connectors/range-slider/connectRangeSlider.js

```diff
@@ -238,7 +238,7 @@ export default function connectRangeSlider(renderFn) {
 
         renderFn(
           {
-            refine: this._refine,
+            refine: this._refine(helper),
```

Comment:
I'm not a super fan to re instantiate a new function on every render, the helper reference won't change in between two calls we can cache this.

Reply:
Later I add also `range` as parameter, both can be cached but I prefer to rely on arguments for function instead of `this._cachedRange`. IMHO it's easier to reason about.